python/GangaDirac/Lib/Files/DiracFile.py

```diff
@@ -463,7 +463,7 @@ def location(self):
                     LFNs.append(this_url)
             return LFNs
 
-    def accessURL(self, thisSE=''):
+    def accessURL(self, thisSE='', protocol='xroot'):
```

Comment:
@rob-c Yes I'll make it so that it is just LHCb that uses xroot. As you say for other collaborations something else may be more appropriate.